js_modules/dagit/packages/core/src/schedules/SchedulesNextTicks.tsx

```diff
@@ -105,7 +105,12 @@ export const SchedulesNextTicks: React.FC<{
         <NonIdealState
           icon="error"
           title="No scheduled ticks"
-          description="There are no running schedules. Start a schedule to see scheduled ticks."
+          description={
+            <>
+              There are no running schedules. <Link to="/overview/schedules">Start a schedule</Link>{' '}
+              to see scheduled ticks.
+            </>
```

Comment:
I'm thinking there are two cases here that we can show, since we already fetch the schedules anyway:

- If there are no running schedules, but some schedules exist, then they should visit the schedules tab to start a schedule
- If there are no schedules at all in any of the repositories, then we can link to the schedule documentation?